pages/common/mosh.md

```diff
@@ -0,0 +1,33 @@
+# mosh
+
+> Mobile Shell (`mosh`) is remote terminal application that allows roaming.
+> `mosh` is a more robust and responsive replacement for SSH.
+> Visit <https://mosh.org> for more information.
+
+- Typical usage:
+
+`mosh {{username}}@{{remote_host}}`
+
+- Usage when `mosh-server` binary is outside standard path:
```

Comment:
This is definitely a good use-case, since `mosh` (both the server and client components) can be installed by an unprivileged user.